FlagsmithClient/Classes/Traits.swift

```diff
@@ -14,10 +14,19 @@ public struct Traits: Codable, Sendable {
     public let traits: [Trait]
     public let identifier: String?
     public let flags: [Flag]
+    public let transient: Bool
     
-    init(traits: [Trait], identifier: String?, flags: [Flag] = []) {
+    init(traits: [Trait], identifier: String?, flags: [Flag] = [], transient: Bool? = false) {
```

Comment:
Do we need the `?` here? We don't use it [here](https://github.com/Flagsmith/flagsmith-ios-client/pull/68/files#diff-e514609a70fb46427745ce4938324decdf3158ec605ea559ee9ce3a6c38f88c4R88), for example.

Reply:
Fixed.